microsoft-edge/webview2/release-notes.md

```diff
@@ -105,27 +105,131 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
-## 1.0.2045.28
+## 1.0.2106-prerelease
 
-Release Date: September 18, 2023
+Release Date: September 19, 2023
 
-[NuGet package for WebView2 SDK 1.0.2045.28](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2045.28)
+[NuGet package for WebView2 SDK 1.0.2106-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2106-prerelease)
 
-For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 117.0.2045.28 or higher.
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 119.0.2106.0 or higher.
 
 
+#### Experimental features
+
+
+<!-- ------------------------------ -->
+* For WinRT, options have been added to manage custom scheme registration when creating a `CoreWebView2Environment`:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+n/a; WinRT only
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2CustomSchemeRegistration` Class
+    * [CoreWebView2CustomSchemeRegistration.AllowedOrigins Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2customschemeregistration?view=webview2-winrt-1.0.2106-prerelease&preserve-view=true#allowedorigins)
+    * [CoreWebView2CustomSchemeRegistration.SchemeName Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2customschemeregistration?view=webview2-winrt-1.0.2106-prerelease&preserve-view=true#schemename)
+* `CoreWebView2EnvironmentOptions` Class
+    * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.2106-prerelease&preserve-view=true#customschemeregistrations)
+
+##### [Win32/C++](#tab/win32cpp)
+
+n/a; WinRT only
+
+---
+
+
+<!-- ------------------------------ -->
+* Added the Frame Process Info API, enabling host applications to identify which processes are for each WebView2 control, to assess the performance impact of WebView2 on the application and on the user's device.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+<!-- todo: add for .NET, like for Win32?
+* `CoreWebView2Environment` Class
+    * [CoreWebView2Environment.GetProcessExtendedInfos Method]()
+* [CoreWebView2FrameInfo Class]()
+    * [CoreWebView2FrameInfo.ParentFrameInfo Property]()
+    * [CoreWebView2FrameInfo.FrameId Property]()
+    * [CoreWebView2FrameInfo.FrameKind Property]()
+-->
+* `CoreWebView2ProcessExtendedInfo` Class
+    * [CoreWebView2ProcessExtendedInfo.AssociatedFrameInfos Property](/dotnet/api/microsoft.web.webview2.core.corewebview2processextendedinfo.associatedframeinfos?view=webview2-dotnet-1.0.2106-prerelease&preserve-view=true)
+    * [CoreWebView2ProcessExtendedInfo.ProcessInfo Property](/dotnet/api/microsoft.web.webview2.core.corewebview2processextendedinfo.processinfo?view=webview2-dotnet-1.0.2106-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+<!-- todo: add for WinRT, like for Win32?

```

Comment:
@victorhuangwq @peiche-jessica 
Should WinRT tab contain these links, like Win32 has?
* `CoreWebView2Environment` Class
    * [CoreWebView2Environment.GetProcessExtendedInfos Method]()
* [CoreWebView2FrameInfo Class]()
    * [CoreWebView2FrameInfo.ParentFrameInfo Property]()
    * [CoreWebView2FrameInfo.FrameId Property]()
    * [CoreWebView2FrameInfo.FrameKind Property]()

When API Ref PR exists,
https://github.com/MicrosoftDocs/webview2-winrt-reference/pulls
i can investigate

Reply:
added